src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -419,8 +433,33 @@ export default function DeploymentConfig({
         }
     }
 
-    async function handleSubmit(e) {
+    const closeLockedDiffDrawerWithChildModal = () => {
+        state.showConfirmation && closeConfirmationDialog()
+        state.showSaveChangsModal && toggleSaveChangesModal()
+        dispatch({
+            type: DeploymentConfigStateActionTypes.toggleShowLockedTemplateDiff,
+            payload: false,
+        })
```

Comment:
extract this out as a separate function `close LockedDiffDrawer`

Reply:
closing lockDiff drawer not required separately anywhere here so can be skipped